drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -0,0 +1,649 @@
+# Nordic Wi-Fi driver for nRF700x
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig WIFI_NRF70
+	bool "nRF700x driver"
+	select NET_L2_WIFI_MGMT if NETWORKING
+	select NET_L2_ETHERNET_MGMT if NETWORKING && NET_L2_ETHERNET
+	select WIFI_USE_NATIVE_NETWORKING if NETWORKING
+	select EXPERIMENTAL if !SOC_SERIES_NRF53X && !SOC_SERIES_NRF91X
+	help
+	  Nordic Wi-Fi Driver
+
+if WIFI_NRF70
+module = WIFI_NRF70_BUS
+module-dep = LOG
+module-str = Log level for Wi-Fi nRF700x bus layers
+module-help = Sets log level for Wi-Fi nRF700x bus layers
+source "subsys/net/Kconfig.template.log_config.net"
+
```

Comment:
Please add
```
config HEAP_MEM_POOL_ADD_SIZE_NRF70
	int
	default 130000
```
here, the special `HEAP_MEM_POOL_ADD_SIZE_` name tells zephyr to add 130000 bytes to the heap size, instead of requiring the sample/app to manually add heap pool sizes.

Reply:
I had tried this option, and used `def_int 130000`, but as I said this is not just dependent on the driver, but depends on sample configuration (network buffers, driver queue len, driver packet buffers etc).

We can still have some default in the driver, but it would need an override in the sample if config changes.

Reply:
> We can still have some default in the driver, but it would need an override in the sample if config changes.

This is expected. we need to over estimate here to make sure the default will run. Optimizing the heap (decreasing the HEAP size for a specific configuration overwriting the default here) is done by the user at a later stage against a complete configuration :)